java-shared-config/pom.xml

```diff
@@ -658,5 +668,36 @@
         <maven.compiler.release>8</maven.compiler.release>
       </properties>
     </profile>
+    <profile>
+      <!-- Profile to use Airlock (go/airlock/howto_maven). Disabled by default. -->
+      <id>airlock-trusted</id>
+      <repositories>
+        <repository>
+          <id>airlock</id>
+          <name>Airlock</name>
+          <url>artifactregistry://us-maven.pkg.dev/artifact-foundry-prod/maven-3p-trusted</url>
+          <layout>default</layout>
+          <releases>
+            <enabled>true</enabled>
+          </releases>
+          <snapshots>
+            <enabled>true</enabled>
```

Comment:
Why do we need to enable snapshot repo?

Reply:
It's following the recommended configuration in go/airlock/howto_maven#maven-configuration.